src/Functions/hasAnyAllTokens.cpp

```diff
@@ -446,6 +615,59 @@ SELECT count() FROM table WHERE hasAllTokens(msg, tokens('a()d', 'splitByString'
         R"(
 ┌─count()─┐
 │       1 │
+└─────────┘
+        )"
+    },
+    {
+        "Usage examples for array and map columns",
+        R"(
+CREATE TABLE log (
+    id UInt32,
+    tags Array(String),
+    attributes Map(String, String),
+    INDEX idx_tags (tags) TYPE text(tokenizer = splitByNonAlpha),
+    INDEX idx_attributes_keys mapKeys(attributes) TYPE text(tokenizer = array),
```

Comment:
Another question here: How do you expect that this syntax could be updated with the preprocessor? Some ideas?